Dockerfile

```diff
@@ -7,7 +7,7 @@
 # envoy-binary pulls in the latest Envoy binary, as Envoy don't publish
 # prebuilt binaries in any other form.
 ARG GOLANG_VERSION
-FROM envoyproxy/envoy-distroless:v1.26.8 as envoy-binary
+FROM envoyproxy/envoy-distroless:v1.27.5 as envoy-binary
```

Comment:
Reviewer note: `envoy-fips` did not get introduced until 1.2.x, so is omitted here